iOS/Source/InformationButton.swift

```diff
@@ -47,8 +47,8 @@ class InformationButton: UIButton {
         self.nameLabel.heightAnchor.constraint(equalToConstant: 22).isActive = true
     }
 
-    func updateInterface(withBackgroundColor backgroundColor: UIColor, andTextColor textColor: UIColor) {
-        self.sun.tintColor = textColor.withAlphaComponent(0.6)
-        self.nameLabel.textColor = textColor.withAlphaComponent(0.6)
+    func updateInterface(controller: DaylightModelController) {
+        self.sun.tintColor = controller.secondaryColor.withAlphaComponent(0.6)
+        self.nameLabel.textColor = controller.secondaryColor.withAlphaComponent(0.6)
```

Comment:
Is the `withAlphaComponent` bit something that should still be happening at this level, or should the controller be handling that?

Reply:
Ah yeah i made a method for that already :V: